RELEASE_NOTES

```diff
@@ -1,14 +1,20 @@
 3.6.0
   - New API
     - ConnectionFactory now also may customize connection closing
-    - GenericTypes.findGenericParameter(Type, Class) now also takes an index, e.g. to resolve `V` in `Map<K, V>`
+    - GenericTypes.findGenericParameter(Type, Class) now also takes an index, e.g. to resolve `V`
+      in `Map<K, V>`
   - New beta API
     - Type qualifiers for binding and mapping. Use annotations to distinguish between different SQL
       data types that map to the same Java type. e.g. VARCHAR, NVARCHAR, and Postgres MACADDR all
       map to String, but are bound and mapped with different JDBC APIs.
     - Support for NVARCHAR columns, using the @NVarchar qualifying annotation
     - Support for Postgres MACADDR columns, using the @MacAddr qualifying annotation.
     - Support for HSTORE columns, using the @HStore annotation
+  - Discontinued API
+    - Due to ongoing stability problems arising to Oracle's Maven servers, development on the
+      jdbi3-oracle12 artifact has been discontinued, sources removed from the
+      project, and the earth salted for good measure. For posterity, jdbi3-bom dependency
+      management pins jdbi3-oracle12 to version 3.5.2.
```

Comment:
We can release it separately, so we don't necessarily have to call it "discontinued", we can just treat it as a totally separate artifact.

Reply:
Updated